packages/webgpu/cpp/rnwgpu/api/GPUCanvasContext.cpp

```diff
@@ -23,41 +23,98 @@ void GPUCanvasContext::configure(
   if (!conv(surfaceConfiguration.usage, configuration->usage)) {
     throw std::runtime_error("Error with SurfaceConfiguration");
   }
-  surfaceConfiguration.width = _canvas->getWidth();
-  surfaceConfiguration.height = _canvas->getHeight();
-  _instance.Configure(&surfaceConfiguration);
-  _lastConfig = configuration;
-  _width = _canvas->getWidth();
-  _height = _canvas->getHeight();
+  auto width = _canvas->getWidth();
+  auto height = _canvas->getHeight();
+  surfaceConfiguration.width = width;
+  surfaceConfiguration.height = height;
+  _surfaceConfiguration = surfaceConfiguration;
+  _offscreenSurface->configure(_surfaceConfiguration);
```

Comment:
Check if we need to configure the offscreen (e.g. if when calling configure, the onscreen surface is already available)